backend/src/main/java/com/crm/dto/response/CustomerResponse.java

```diff
@@ -7,7 +7,7 @@
 @Builder
 @EqualsAndHashCode
 @NoArgsConstructor
-@AllArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
```

Comment:
I chyba tej

Reply:
To nie zadziała z Builderem, zostawiłem jak było